process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+# Becoming an OpenSSF Project
+
+The OpenSSF's mission is to inspire and enable the community to secure the open source software we all depend on. OpenSSF's top priority is the security of existing open source projects however, there are missing pieces in the tooling needed to improve open source project security. These pieces are the projects OpenSSF aims to steward and support.
+
+## Project Life Cycle
+
+New projects to the OpenSSF, and progression through the project lifecycle, are approved by the Technical Advisory Committee (TAC). A project's oversight is provided by either the Technical Advisory Committee (TAC) or a specific Working Group (WG). When a Project reports into a specific WG, that WG can support the Project's progression and provide recommendations to the TAC.
+
+Projects follow the Sandbox, Incubating, Graduated, and Archived lifecycle stages defined below. Projects that seek widespread adoption and end user use are expected to progress through the stages. Projects coming to OpenSSF that already meet the entry requirements may enter the Incubating stage directly.
+
+<!-- TOC -->
+
+  - [Stages - Definitions & Expectations](#stages---definitions--expectations)
+      - [Sandbox](#sandbox)
+      - [Incubating](#incubating)
+      - [Graduated](#graduated)
+      - [Archived](#archived)
+
+<!-- /TOC -->
```

Comment:
The picture process/incubation-process.png doesn't seem to be consistent (and I can't comment on a .png). It mentions a "TOC" - what's a TOC? Is that the TAC? The .png seems disconnected.

Reply:
I think TOC here means table of contents.

Reply:
So, actually the "TOC" in the comments above indeed stands for "Table of Contents" but the TOC in the process picture stands for "Technical Oversight Committee". This is because the image comes from the CNCF which has a TOC instead of a TAC. It was committed by @betarelease in the copy of the CNCF files he made to get us started. The original image can be found at: https://github.com/cncf/toc/blob/main/process/incubation-process.png
Unfortunately I don't see any source file for this image. Maybe @[lizrice](https://github.com/lizrice) has it since she has committed that file to their repo?
As a side note: I think it is best to commit along an image the source file used to produce it so that is it easier to modify later on if needed.

Reply:
I've recreated the image in[ this Google Slide deck specific to the TAC](https://docs.google.com/presentation/d/1dwxJ_IJuzHF-mRKO6cvp5f6CvsYe2QPED5VJRz7KBTA/edit?usp=sharing).  It can be leveraged as a stop gap until a Design team manages it.
<img width="919" alt="TAC-Project-Process-AUG2022" src="https://user-images.githubusercontent.com/33327273/182884683-cbf0ebdb-7c6f-479f-92a3-24c1398adb29.png">